Modules/CLI/FiberTractMeasurements/Testing/CompareTxts.cxx

```diff
@@ -26,36 +50,83 @@ int main( int argc, char * argv[] )
   std::ifstream resultTxt(resultTxtPath.c_str());
   std::ifstream baselineTxt(baselineTxtPath.c_str());
 
-  std::string resultLine;
-  std::string baselineLine;
-  std::string resultMeasure;
-  std::string baselineMeasure;
-  int c = 1;
-  while (std::getline(resultTxt, resultLine) && std::getline(baselineTxt, baselineLine))
-   {
-      if (c > 1)
+  auto fileExists = doesFileExist(baselineTxtPath);
+  fileExists &= doesFileExist(resultTxtPath);
+  if (!fileExists)
+    {
+      std::cerr << "Test failed." << std::endl;
+      return EXIT_FAILURE;
+    }
```

Comment:
May be interested in reviewing how we handle check vs exit code:
* https://github.com/Slicer/Slicer/blob/main/Libs/MRML/Core/vtkMRMLCoreTestingMacros.h
* https://github.com/Slicer/Slicer/blob/main/Libs/MRML/Core/vtkMRMLCoreTestingUtilities.h
* https://github.com/Slicer/vtkAddon/blob/main/vtkAddonTestingUtilities.h
* https://github.com/Slicer/vtkAddon/blob/main/vtkAddonTestingMacros.h

Reply:
Thanks, but testing these things should not require using `vtkAddon`, and not sure how the core testing macros and utilities can avoid boilerplate code here.